src/GRA.Controllers/MissionControl/ChallengesController.cs

```diff
@@ -435,12 +434,20 @@ public async Task<IActionResult> Edit(int id)
                 Groups = await _challengeService.GetGroupsByChallengeId(challenge.Id),
                 RelatedEvents = await _eventService.GetByChallengeIdAsync(challenge.Id),
                 BadgeMakerUrl = GetBadgeMakerUrl(siteUrl, site.FromEmailAddress),
-                UseBadgeMaker = true
+                UseBadgeMaker = true,
+                IgnorePointLimits = UserHasPermission(Permission.IgnorePointLimits),
+                MaxPointLimit = await _challengeService.GetMaximumAllowedPointsAsync(site.Id)
             };
 
-            if (site.MaxPointsPerChallengeTask.HasValue)
+            if (challenge.TasksToComplete.HasValue)
             {
-                viewModel.MaxPointsMessage = $"(Up to {site.MaxPointsPerChallengeTask.Value} points per required task)";
+                viewModel.MaxPointLimit = await _challengeService.GetMaximumAllowedPointsAsync(site.Id);
+
+                if (challenge.PointsAwarded / challenge.TasksToComplete.Value
+                    > viewModel.MaxPointLimit)
+                {
+                    viewModel.MaxPointsMessage = "This Challenge's number of points awarded is higher than what is allowed by the configuration.";
```

Comment:
The max points message is an informational one by the label instead of an error message on the field. It should be changed back and only shown if there's any max point limit.

Reply:
Resolved in commit cf14cfc82e0465695f57fa374ec9a4445e055647